app.js

```diff
@@ -1,107 +1,23 @@
-require('dotenv').config()
-const express = require('express')
-const { Pool } = require('pg')
-const app = express()
+require('dotenv').config();
+const express = require('express');
+const { Pool } = require('pg');
+const app = express();
+const { misc, rides, zagster } = require('./routes');
```

Comment:
@Otis0620 But I thought going semi-colon free was the cool modern way. :)

Reply:
No... Always use semi colons. It's NOT cool if they are not there